index.d.ts

```diff
@@ -51,6 +51,7 @@ export interface ToastProviderProps {
     };
     placement?: Placement;
     transitionDuration?: number;
+    portalTargetSelector: string;
```

Comment:
This should be optional, right?

```suggestion
    portalTargetSelector?: string;
```

Reply:
Obv., sorry!